packages/components/src/color-picker/index.native.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * External dependencies
+ */
+import { View, Text, TouchableWithoutFeedback, Platform } from 'react-native';
+import HsvColorPicker from 'react-native-hsv-color-picker';
+import tinycolor from 'tinycolor2';
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { BottomSheet } from '@wordpress/components';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import { Icon, check, close } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+function ColorPicker( {
+	shouldEnableBottomSheetScroll,
+	shouldSetBottomSheetMaxHeight,
+	isBottomSheetScrolling,
+	setBackgroundColor,
```

Comment:
I think it could be refactored as well in the same way as described in my comment here https://github.com/WordPress/gutenberg/pull/21326/files#r408281223